openmc/data/multipole.py

```diff
@@ -391,6 +1017,130 @@ def from_hdf5(cls, group_or_filename):
 
         return out
 
+    @classmethod
+    def from_endf(cls, endf_file, log=False, vf_options=None, wmp_options=None):
+        """Generate windowed multipole neutron data from an ENDF evaluation.
+
+        Parameters
+        ----------
+        endf_file : str
+            Path to ENDF evaluation
+        log : bool or int, optional
+            Whether to print running logs (use int for verbosity control)
+        vf_options : dict, optional
+            Dictionary of keyword arguments, e.g. {'njoy_error': 0.001},
+            passed to :func:`openmc.data.multipole.vectfit_nuclide`
+        wmp_options : dict, optional
+            Dictionary of keyword arguments, e.g. {'search': True, 'rtol': 0.01},
+            passed to :func:`openmc.data.WindowedMultipole.from_multipole`
+
+        Returns
+        -------
+        openmc.data.WindowedMultipole
+            Resonant cross sections represented in the windowed multipole
+            format.
+
+        """
+
+        if vf_options is None:
+            vf_options = {}
+
+        if wmp_options is None:
+            wmp_options = {}
+
+        if log:
+            vf_options.update(log=log)
+            wmp_options.update(log=log)
+
+        # generate multipole data from EDNF
+        mp_data = vectfit_nuclide(endf_file, **vf_options)
+
+        # windowing
+        return cls.from_multipole(mp_data, **wmp_options)
+
+    @classmethod
+    def from_multipole(cls, mp_data, search=None, log=False, **kwargs):
+        """Generate windowed multipole neutron data from multipole data.
+
+        Parameters
+        ----------
+        mp_data : dictionary or str
+            Dictionary or Path to the multipole data stored in a pickle file
+        search : bool, optional
+            Whether to search for optimal window size and curvefit order
```

Comment:
Since the default is now hidden in the code, the description here should describe what the default behavior is

Reply:
comments added